### PR TITLE
Ports "Environment Protection Bags + Minor Weather Changes" and others

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Ports four PRs:

- ["BSbodybags inherit heaviest item weight + Easier to escape the disarm/BSbodybag combo" #54065](https://github.com/tgstation/tgstation/pull/54065) by ExcessiveUseOfCobblestone
- ["Environment Protection Bags + Minor Weather Changes"](https://github.com/tgstation/tgstation/pull/59752) #59752 by Greniza
- ["Weather runtime & EPB fix" #60043](https://github.com/tgstation/tgstation/pull/60043) by Greniza
- ["[NONMODULAR] Adds TG's "Environment Bodybag" to the mining vendor" #7874](https://github.com/Skyrat-SS13/Skyrat-tg/pull/7874) by RatFromTheJungle, though tweaked as to not make shelters redundant

Also gives the mining medic a bag of environmental body bags to save miners with (may just change it to one) and added the syndicate body bag to the uplink (may change it to infiltrator/nukies only or increase the price). I can atomize this down if needed.

image
image

# Wiki Documentation

Vending machines, uplink, and mining medic wikipages will need to be updated with the new bags. 

# Changelog

:cl:  Greniza, ExcessiveUseOfCobblestone, RatFromTheJungle
tweak: bluespace bodybags now inherit the weight of its heaviest item and It is much easier for a conscious individual to escape from them.
rscadd: Environment protection bags, which protect their occupants from weather, and give protection from temperature and pressure. The rare variants even have the same protection effects as a space suit.
/:cl:

### CODE REVIEW

**Comment by tattax** [comment]
changes on bs body bag are good aside from the weight change as i don't see a reason why

**Comment by nmajask** [comment]
> changes on bs body bag are good aside from the weight change as i don't see a reason why  

Main reason I ported that PR was because it refactored some of the code for body bags and I would have needed to rewrite a bunch of code for the environmental protection bags, but I still think it's a good change. Right now, BS body bags can be used to store a ton of items and make inventory management only matter for quick accessibility, which I don't think is the best. I can remove/atomize it out if people want.

**Comment by KillerOrcaCora** [comment]
Very based

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -139,6 +139,8 @@
 	name = "\improper Rod of Asclepius"
 	desc = "A wooden rod about the size of your forearm with a snake carved around it, winding its way up the sides of the rod. Something about it seems to inspire in you the responsibilty and duty to help others."
 	icon = 'icons/obj/lavaland/artefacts.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/staves_lefthand.dmi'
```
Comment (SomeguyManperson) [inline code/modules/mining/lavaland/necropolis_chests.dm]: e

Comment (nmajask) [reply code/modules/mining/lavaland/necropolis_chests.dm]: fixed

**Review by nesisknesis — APPROVED** [review]

**Pull request merged**